### DISCUSSION TITLE
Is there a way to share an album with download disabled ?

### DISCUSSION THREAD

**Discussion by Tiritibambix** [photoprism/photoprism#4101] [Q&A]
Hello, I'm new to photoprism and it looks great.

I was trying things here and there and saw the share album function.
I see I can set an expiration date for the link, customize the slug, but there is no option to disable download per album. I believe this is a general function.
Sometimes, I just want some people to see my pictures, but I don't want them to be able to download them.?

Am I missing something or is it just not possible to disable download for specific albums ?

**Comment by graciousgrey** [comment]
Hello :wave: 

That is not possible at the moment.

We may add an option in the future to not show the download buttons on shared links. However, this would only prevent the original files from being downloaded. Thumbnails could still be downloaded using the browser's built-in functionalities.

**Comment by Tiritibambix** [comment]
To be honest, I don't really mind if people go the extra mile to download thumbnails. My main focus is on them being able to download full-size photos or not.

Just to be open with you, I used to use Lychee and it was alright. But now with version 5 acting up and basically not working, I'm on the lookout for a new option. The Lychee team has admitted they don't have the resources to fix these issues.

I'm in the process of starting a photography business and I need something reliable. I checked out Photoprism a while back and thought it was still developing, but after giving it another try, I think it might be a good fit for what I need.

Specifically, I am hopeful to see the following features incorporated into Photoprism:

- Hidden: Anonymous users need a direct link to access this album.
- Downloadable: Anonymous users can download this album.
- Original: Anonymous users can behold full-resolution photos.
- Password protected: Anonymous users need a shared password to access this album.

I'm really looking forward to reading your answer on this topic !

**Comment by graciousgrey** [comment]
Thank you for your feedback!

Hidden: Anonymous users need a direct link to access this album.

 and 

Downloadable: Anonymous users can download this album.

are already possible. You can share a link that can be accessed by anonymous users without a password. The original photos can be downloaded with the download action.

https://docs.photoprism.app/user-guide/share/

Password protection for shared links and many other improvements are planned. However, we can't tell you yet when we'll get to this.

**Comment by Tiritibambix** [comment]
Thank you for your answer.

> Downloadable: Anonymous users can download this album.

One suggestion to consider here is the option to disable album downloads, but on a per album basis.

I'm super excited about the improvements you mentioned! I definitely prefer subscribing to Photoprism over pic-time or any other closed source software. Can't wait to see how Photoprism progresses! 😊
